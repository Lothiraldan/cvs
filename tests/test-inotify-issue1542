#!/bin/sh

"$TESTDIR/hghave" inotify || exit 80

hg init

touch a
mkdir dir
touch dir/b
touch dir/c

echo "[extensions]" >> $HGRCPATH
echo "inotify=" >> $HGRCPATH

hg add dir/c

echo % inserve
hg inserve -d --pid-file=hg.pid 2>&1
cat hg.pid >> "$DAEMON_PIDS"

hg st

echo % moving dir out
mv dir ../tmp-test-inotify-issue1542

echo % status
hg st

sleep 1
echo "Are we able to kill the service? if not, the service died on some error"
kill `cat hg.pid`

