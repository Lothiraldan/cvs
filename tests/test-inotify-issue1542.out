% inserve
A dir/c
? a
? dir/b
? hg.pid
% moving dir out
% status
! dir/c
? a
? hg.pid
Are we able to kill the service? if not, the service died on some error
